evals/mcpchecker/README.md

```
@@ -5,8 +5,8 @@ Evaluations for obs-mcp using [mcpchecker](https://github.com/mcpchecker/mcpchec
 ## Pre-requisites
 
 - [mcpchecker](https://github.com/mcpchecker/mcpchecker#install) installed (v0.0.16+) — run `make install-mcpchecker` from the repo root
-- A Kubernetes/OpenShift cluster with Prometheus and Alertmanager running
-- obs-mcp server deployed and accessible (see [Backend Setup](#backend-setup))
+- **Metrics / alerts / traces / otelcol:** Kubernetes or OpenShift cluster with Prometheus and Alertmanager (see [Backend Setup](#backend-setup))
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)